sdk/personalizer/azure-ai-personalizer/src/main/java/com/azure/ai/personalizer/PersonalizerAdminAsyncClient.java

```diff
@@ -0,0 +1,604 @@
+import com.azure.ai.personalizer.models.PersonalizerServiceProperties;
+import com.azure.core.annotation.ReturnType;
+import com.azure.core.annotation.ServiceClient;
+import com.azure.core.annotation.ServiceMethod;
+import com.azure.core.http.rest.PagedFlux;
+import com.azure.core.http.rest.PagedResponse;
+import com.azure.core.http.rest.PagedResponseBase;
+import com.azure.core.http.rest.Response;
+import com.azure.core.http.rest.SimpleResponse;
+import com.azure.core.util.BinaryData;
+import com.azure.core.util.Context;
+import com.azure.core.util.CoreUtils;
+import com.azure.core.util.FluxUtil;
+import com.azure.core.util.logging.ClientLogger;
+import reactor.core.publisher.Mono;
+
+import java.util.Objects;
+
+import static com.azure.core.util.FluxUtil.monoError;
+import static com.azure.core.util.FluxUtil.withContext;
+
+/**
+ * Client to perform administrative operations on Personalizer instance in an asynchronous manner.
+ */
+@ServiceClient(builder = PersonalizerClientBuilder.class, isAsync = true)
+public final class PersonalizerAdminAsyncClient {
+
+    private final PersonalizerClientV1Preview3Impl service;
+    private final ClientLogger logger = new ClientLogger(PersonalizerAdminAsyncClient.class);
+
+    PersonalizerAdminAsyncClient(PersonalizerClientV1Preview3Impl service) {
+        this.service = service;
+    }
+
+    /**
+     * Submit a new Offline Evaluation job.
+     * @param evaluationOptions The Offline Evaluation job definition.
+     * @return a counterfactual evaluation on successful completion of {@link Mono}.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<PersonalizerEvaluation> createEvaluation(PersonalizerEvaluationOptions evaluationOptions) {
+        return createEvaluationWithResponse(evaluationOptions).flatMap(FluxUtil::toMono);
+    }
+
+    /**
+     * Submit a new Offline Evaluation job.
+     * @param evaluationOptions The Offline Evaluation job definition.
+     * @return a counterfactual evaluation along with {@link Response} on successful completion of {@link Mono}.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Response<PersonalizerEvaluation>> createEvaluationWithResponse(PersonalizerEvaluationOptions evaluationOptions) {
+        try {
+            return withContext(context -> createEvaluationWithResponse(evaluationOptions, context));
+        } catch (RuntimeException ex) {
+            return monoError(logger, ex);
+        }
+    }
+
+    /**
+     * Submit a new Offline Evaluation job.
+     * @param evaluationOptions The Offline Evaluation job definition.
+     * @param context The context to associate with this operation.
+     * @return a counterfactual evaluation along with {@link Response} on successful completion of {@link Mono}.
+     */
+    Mono<Response<PersonalizerEvaluation>> createEvaluationWithResponse(PersonalizerEvaluationOptions evaluationOptions, Context context) {
```

Comment:
>createEvaluationWithResponse [](http://example.com/codeflow?start=43&length=28)

From offline chat with Sameeksha, the ask is to make this a long running operation that returns PollerFlux.